eng/scripts/Create-ApiReview.ps1

```diff
@@ -0,0 +1,185 @@
+
+    if ($packages)
+    {
+        foreach($pkgPath in $packages.Values)
+        {
+            $pkg = Split-Path -Leaf $pkgPath
+            $pkgPropPath = Join-Path -Path $ConfigFileDir "$PackageName.json"
+            if (-Not (Test-Path $pkgPropPath))
+            {
+                Write-Host " Package property file path $($pkgPropPath) is invalid."
+                continue
+            }
+            # Get package info from json file created before updating version to daily dev
+            $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
+            $version = [AzureEngSemanticVersion]::ParseVersionString($pkgInfo.Version)
+            if ($version -eq $null)
+            {
+                Write-Host "Version info is not available for package $PackageName, because version '$(pkgInfo.Version)' is invalid. Please check if the version follows Azure SDK package versioning guidelines."
+                return 1
+            }
+
+            Write-Host "Version: $($version)"
+            Write-Host "SDK Type: $($pkgInfo.SdkType)"
+            Write-Host "Release Status: $($pkgInfo.ReleaseStatus)"
+
+            # Run create review step only if build is triggered from main branch or if version is GA.
+            # This is to avoid invalidating review status by a build triggered from feature branch
+            if ( ($SourceBranch -eq $DefaultBranch) -or (-not $version.IsPrerelease))
+            {
+                Write-Host "Submitting API Review for package $($pkg)"
+                $respCode = Submit-APIReview -reviewFileName $reviewFileName -packageArtifactname $pkg -apiLabel $($pkgInfo.Version) -releaseStatus $pkgInfo.ReleaseStatus 
+                Write-Host "HTTP Response code: $($respCode)"
+                # HTTP status 200 means API is in approved status
+                if ($respCode -eq '200')
+                {
+                    Write-Host "API review is in approved status."
+                }
+                elseif ($version.IsPrerelease)
+                {
+                    # Ignore API review status for prerelease version
+                    Write-Host "Package version is not GA. Ignoring API view approval status"
+                }
+                elseif (!$pkgInfo.ReleaseStatus -or $pkgInfo.ReleaseStatus -eq "Unreleased")
+                {
+                    Write-Host "Release date is not set for current version in change log file for package. Ignoring API review approval status since package is not yet ready for release."
+                }
+                else
+                {
+                    # Return error code if status code is 201 for new data plane package
+                    # Temporarily enable API review for spring SDK types. Ideally this should be done be using 'IsReviewRequired' method in language side
+                    # to override default check of SDK type client
+                    if (($pkgInfo.SdkType -eq "client" -or $pkgInfo.SdkType -eq "spring") -and $pkgInfo.IsNewSdk)
```

Comment:
I assume this is intended to be the common copy which is why you have "spring" here.

Reply:
Correct. Once all T1 languages are moved to sandboxing I will switch this script to eng/common